### PR TITLE
PHPC-1848: Add 5.0 to the server testing matrix

### PR DESCRIPTION
**Pull request by alcaeus**
PHPC-1848

This also updates the config file to use a `latest-stable` alias to avoid having to update multiple build variants when it changes.

### CODE REVIEW

File: tests/manager/manager-executeQuery_error-003.phpt [hunk 1]
```diff
@@ -24,7 +24,7 @@ try {
 ===DONE===
 <?php exit(0); ?>
 --EXPECT--
-MongoDB\Driver\Exception\CommandException(2): unknown top level operator: $foo
```
Comment (jmikola) [inline tests/manager/manager-executeQuery_error-003.phpt:27]: Why was this change needed? Testing with 5.0 locally, this error message/code appears to still be returned for a query with `{$foo: 1}`.

While we're changing this, I would suggest using `$unsupportedOperator` to make the test more self-documenting.

**Review by jmikola — APPROVED** [review]
One question about the test change, but LGTM otherwise.

File: tests/manager/manager-executeQuery_error-003.phpt (hunk 1 above)
Comment (alcaeus) [reply tests/manager/manager-executeQuery_error-003.phpt:27]: Here's a diff from the test run:
```diff
- BulkWriteException: unknown top level operator: $foo
+ BulkWriteException: unknown top level operator: $foo. If you have a field name that starts with a '$' symbol, consider using $getField or $setField.
```

I'll use `$unsupportedOperator` instead.

Comment (jmikola) [reply tests/manager/manager-executeQuery_error-003.phpt:27]: Oh, interesting. That must have been added in the rc0 release. I was still running an earlier 5.0 version locally.
